editor/code_editor.cpp

```diff
@@ -815,7 +815,7 @@ void CodeTextEditor::input(const Ref<InputEvent> &event) {
 void CodeTextEditor::_text_editor_gui_input(const Ref<InputEvent> &p_event) {
 	Ref<InputEventMouseButton> mb = p_event;
 
-	if (mb.is_valid()) {
+	if (mb.is_valid() && zoom_shortcut_enabled) {
 		if (mb->is_pressed() && mb->is_command_or_control_pressed()) {
 			if (mb->get_button_index() == MouseButton::WHEEL_UP) {
 				_zoom_in();
```

Comment:
The change seems correct, but I'm puzzled by the pre-existing code.

Why does it check the mouse wheel input manually here, and then also checks the `script_editor/zoom_in/out` shortcuts below?

CC @Paulb23 

If this was standardized on checking only the shortcuts, then the new option could in theory not be absolutely needed (one can just remove the default bindings for the shortcuts). It could still be added for convenience if there's support for it of course.

Reply:
The code might have been added before there was a standardized way for checking shortcuts. Also it isn't really possible to change editor keybinds or atleast not obvious how-to.

Reply:
It should be possible, in the "Shortcuts" tab of the Editor Settings dialog.

But indeed the ones declared in this file seem to be missing. That's a bug that needs to be looked into.
![image](https://github.com/godotengine/godot/assets/4701338/006ddb6a-a4ab-4f40-8d16-92ae8315fe6d)

Other script editor shortcuts seem present:
![image](https://github.com/godotengine/godot/assets/4701338/a2d044a4-ee22-41de-800c-d1395040c22d)

Reply:
Ah they're present once a script is actually edited in the script editor:
![image](https://github.com/godotengine/godot/assets/4701338/18c48163-984d-4213-9ca4-bc8368afb116)

Since they run in the constructor of the code editor, which hadn't been instantiated yet when I tested in an empty project. Should probably be fixed somehow.

Reply:
I think the Ctrl + Mousewheel shortcut is independent of the editor settings...

Reply:
Aaanyway, I had a closer look, and now I understand why there's two separate handling of the zoom. This one is for mouse wheel events, while the one below using editor shortcuts is for key events (<kbd>Ctrl</kbd> + <kbd>+</kbd>, <kbd>Ctrl</kbd> + <kbd>-</kbd>).

I suppose there isn't much reason for disabling the latter, so I think the editor setting should only disable mouse wheel zooming, and be named accordingly.

Reply:
So I changed the setting to only affect zooming Ctrl + Mouse wheel. The key shortcuts can simply be disabled in the shorcuts menu.